### ISSUE TITLE
 Replace Bootstrap tour by IntroJS in tutorial

### ISSUE THREAD

**Issue by llaske** [llaske/sugarizer-server#363]
[Bootstrap tour](https://bootstraptour.com/) is the library use for writing tutorial. The library is no longer maintain and doesn't support recent version of Bootstrap. So we decided to replace it by another libraries named [IntroJS](https://introjs.com/).
IntroJS provide similar features and a lightweight UI.

This replacement has be done in Sugarizer app. See [here](https://github.com/llaske/sugarizer/issues/1090) to learn how.

The idea is to replace it in Sugarizer Server Dashboard.

![image](https://user-images.githubusercontent.com/981841/219852088-0fccc378-b411-4eec-b28e-b1eb94706a5d.png)

**Comment by parteekcoder** [comment]
@llaske is this issue still opened as there is one merged PR mentioned here

**Comment by llaske** [comment]
@parteekcoder yes the issue is still open, the PR was not directly related to the issue.

**Comment by rockharshitmaurya** [comment]
hey, @llaske I have fixed this issue in PR:https://github.com/llaske/sugarizer-server/pull/395. Please review it and let me know if any changes are required

**Comment by llaske** [comment]
Thanks @rockharshitmaurya .
There is an issue on the Stats screen.

![image](https://user-images.githubusercontent.com/981841/235771962-af0198e4-10da-4c7d-86c2-55224bcf38d1.png)

Plus the CI/CD fail due to lot if errors in eslint.

![image](https://user-images.githubusercontent.com/981841/235772698-dd120852-6ba2-4eed-90d0-c73be813f706.png)

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer-server/pull/417
